packages/desktop-client/src/components/accounts/Account.tsx

```diff
@@ -1780,6 +1780,15 @@ class AccountInternal extends PureComponent<
                   sortField={this.state.sort?.field}
                   ascDesc={this.state.sort?.ascDesc}
                   onChange={this.onTransactionsChange}
+                  onBatchDelete={this.onBatchDelete}
```

Comment:
There is a `useTransactionBatchActions` hook. Maybe we can use that here?

Reply:
I just tried it. But unfortunatly the table doesn't update quite right. When you duplicate an transaction you need te reload the page to see the change.